microsoft-365/security/office-365-security/mdo-support-teams-about.md

```diff
@@ -30,7 +30,7 @@ ms.date: 3/29/2023
 - [Microsoft Defender for Office 365 plan 2](defender-for-office-365.md)
 
 > [!NOTE]
-> This article lists new features in the latest release of Microsoft Defender for Office 365. These features are currently in preview.
+> This article lists new features in the latest release of Microsoft Defender for Office 365. These features are currently in preview. Once you run the cmdlet, please be aware that it will take a few days for the features to to be available. 
```

Comment:
```suggestion
> This article lists new features in the latest release of Microsoft Defender for Office 365. These features are currently in preview.
>
> Once you run the cmdlet, be aware that it will take a few days for the features to be available. 
```